spec/models/academic_year_spec.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: academic_years
+#
+#  id         :uuid             not null, primary key
+#  ends_on    :date
+#  name       :string
+#  starts_on  :date
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+require "rails_helper"
+
+RSpec.describe AcademicYear, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Likewise, although it seems like the AcademicYear model doesn't have much (any?) logic right now.